files/ko/web/api/rtcpeerconnection/currentlocaldescription/index.md

```diff
@@ -2,11 +2,23 @@
 title: RTCPeerConnection.currentLocalDescription
 slug: Web/API/RTCPeerConnection/currentLocalDescription
 translation_of: Web/API/RTCPeerConnection/currentLocalDescription
+page-type: web-api-instance-property
+tags:
+  - API
+  - Media
+  - Property
+  - RTCPeerConnection
+  - Read-only
+  - Reference
+  - SDP
+  - WebRTC
+  - currentLocalConnection
+browser-compat: api.RTCPeerConnection.currentLocalDescription
 ---
 
-{{WebRTCSidebar}}
+{{DefaultAPISidebar("WebRTC")}}
 
-읽기 속성인 **`RTCPeerConnection.currentLocalDescription`**은 가장 최근에 {{domxref("RTCPeerConnection")}}가 성공적으로 네고시에이션을 마치고 원격 피어와 연결된, 연결인터페이스의 로컬 엔드를 설명하는 {{domxref("RTCSessionDescription")}} 객체를 반환합니다. 이외에도 설명에 의해 offer 혹은 answer가 처음으로 인스턴스화 되면 ICE 에이전트에 의해 이미 생성됬을수도 있는 ICE candidate 목록이 포함되어 있습니다.
```

Comment:
These are usually bolded. There is a general cleanup for all the stuck bolding in the KO docs in https://github.com/mdn/translated-content/pull/10273